psi4/src/psi4/libmints/thc_eri.h

```diff
@@ -97,6 +97,8 @@ class LS_THC_Computer : public THC_Computer {
     SharedMatrix build_E_exact();
     /// Parrish LS-THC Procedure 3
     SharedMatrix build_E_df();
+    /// Matthews 2020 SI Page 5 (returns S matrix using rank-reduced grid)
```

Comment:
I would also put the DOI of the Matthews 2020 paper here, for easy access to developers working on this part of the code.

Reply:
Done!